geoapi/routes/public_projects.py

```diff
@@ -59,6 +60,11 @@ class PublicProjectPointCloudsResource(ProjectPointCloudsResource, metaclass=Hid
     pass
 
 
+@api.route('/<int:projectId>/export/')
```

Comment:
I don't think we need this as we don't want to offer this route to unauthenticated users, right?